samples/bot-feed-members/nodejs/bots/teamsConversationBot.js

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+const {
+    ActionTypes,
+    CardFactory,
+    MessageFactory,
+    TeamsActivityHandler,
+    TeamsInfo,
+    TurnContext
+} = require('botbuilder');
+
+class TeamsConversationBot extends TeamsActivityHandler {
+    constructor() {
+        super();
+
+        this.onMessage(async (context, next) => {
+            // Code to get details of all members in group chat
+            const members = await TeamsInfo.getPagedMembers(context);
+            TurnContext.removeRecipientMention(context.activity);
+
+            const text = context.activity.text.trim().toLocaleLowerCase();
+            if (text.includes('list')) {
+                await this.listMembersAsync(context, members);
+            } else {
+                await this.cardActivityAsync(context, false);
+            }
+
+            await next();
+        });
+
+        this.onMembersAdded(async (context, next) => {
+                const members = await TeamsInfo.getPagedMembers(context);
```

Comment:
should not get all members here, use the member id returned by the event

Reply:
Updated